modules/tensor_mechanics/include/materials/ADLAROMANCEStressUpdateBase.h

```diff
@@ -60,7 +73,8 @@ class ADLAROMCreepStressUpdate : public ADRadialReturnCreepStressUpdateBase<comp
   /// Function to check input values against applicability windows set by ROM data set
   void checkInputWindows(std::vector<ADReal> & input);
 
-  /* Convert the input variables into the form expected by the ROM Legendre polynomials
+  /**
+   * Convert the input variables into the form expected by the ROM Legendre polynomials
```

Comment:
The doxygen descriptions for these methods should define inputs and outputs using `@param` and  `@return`.

Reply:
Boy...I love looking at doxygen, but I hate @param and @return! done for this one though.